client/packages/invoices/src/InboundShipment/DetailView/ContentArea/columns.ts

```diff
@@ -229,15 +220,15 @@ export const useInboundShipmentColumns = () => {
               const { lines } = rowData;
               return ArrayUtils.ifTheSameElseDefault(lines, 'packSize', '');
             } else {
-              return getPackSize(rowData) ?? '';
+              return rowData.packSize ?? '';
```

Comment:
added some of these back in for placeholder lines, to make the view consistent between grouped and not